food-and-agriculture-organization-of-the-united-nations.csl

```diff
@@ -10,196 +10,368 @@
     <author>
       <name>Bin Liu</name>
       <email>lieubean@gmail.com</email>
+      <uri>https://www.linkedin.com/in/lieubean/</uri>
     </author>
+    <contributor>
+      <name>Julian Plummer</name>
+      <email>Julian.Plummer@fao.org</email>
+    </contributor>
     <category citation-format="author-date"/>
     <category field="science"/>
     <category field="social_science"/>
-    <summary>This style is created to meet the citation and bibliographical requirements of FAO House Style.</summary>
-    <updated>2015-08-24T12:56:02+00:00</updated>
+    <summary>This style is created to meet the citation and bibliographical requirements of FAOSTYLE, and has been tested with Zotero and Mendeley. Last update: March 2017.</summary>
+    <updated>2017-03-08T16:33:05+01:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <!-- Locale settings for English. Settings for other languages could be added later to this section. -->
   <locale xml:lang="en">
+    <style-options punctuation-in-quote="false"/>
     <terms>
       <term name="editor" form="verb-short">ed. by</term>
       <term name="editor" form="short">
         <single>ed</single>
         <multiple>eds</multiple>
       </term>
+      <term name="no date">undated</term>
     </terms>
+    <date delimiter=" " form="text">
+      <date-part name="day"/>
+      <date-part name="month"/>
+      <date-part name="year"/>
+    </date>
   </locale>
+  <!-- Macros -->
   <macro name="editor">
-    <names variable="editor" delimiter="," suffix=".">
+    <!-- XX ed. / XX & XX eds. -->
+    <names variable="editor">
       <name and="symbol" delimiter-precedes-last="never" initialize-with="."/>
       <label form="short" strip-periods="false" prefix=", "/>
     </names>
   </macro>
   <macro name="author">
-    <names variable="author" suffix=". ">
-      <name and="symbol" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all"/>
-      <label form="short" prefix=" "/>
-      <substitute>
-        <names variable="editor"/>
-        <names variable="translator"/>
-        <text variable="title"/>
-      </substitute>
-    </names>
+    <choose>
```

Comment:
Need help here. I write the substitute part as:
```
<substitute>
  <names variable="editor"/>
  <names variable="translator"/>
  <text term="anonymous" text-case="capitalize-first"/>
</substitute>
```
But the result is if there's no author/editor/translator, "Anonymous" is not shown. The author part is just missing. How to fix?